package.json

```diff
@@ -43,7 +43,7 @@
     "url": "git+https://github.com/bootstrap-vue/bootstrap-vue.git"
   },
   "files": [
-    "lib",
+    "src",
     "dist",
     "nuxt"
```

Comment:
I think `nuxt` has to change to `docs`

Reply:
This nuxt refers to nuxt module not docs. I will deprecate `@nuxtjs/bootstrap-vue` in flavor of this internal so we can offer an always update integration module :)

Reply:
Ah, ok...